.gitignore

```diff
@@ -21,3 +21,4 @@ docs/build/
 .coverage
 cover
 env
+.vscode
```

Comment:
This should be in your system's global .gitignore.

It does not make sense to add every single file created by every single editor in the world to every project's gitignore.

Reply:
Okey, thanks for link!